azure-pipelines.yml

```diff
@@ -174,18 +174,38 @@ jobs:
           trigger_sha: '$(trigger_sha)'
       - template: ci/report-end.yml
 
-  - job: compatibility
+  - job: compatibility_linux
     dependsOn:
       - check_for_release
     timeoutInMinutes: 60
     pool:
-      name: 'linux-pool'
+      name: linux-pool
+    steps:
+      - template: ci/report-start.yml
+      - checkout: self
+      - template: ci/compatibility.yml
+        parameters:
+          test_flags: '--quick'
+          os: linux
+          push_generic: true
+      - template: ci/tell-slack-failed.yml
+      - template: ci/report-end.yml
+
+  - job: compatibility_macos
```

Comment:
Shouldn’t we also enable macos on the daily compat cron job?

Reply:
Good catch, thanks. I had thought of that at some point but got lost in YAML.